python/random_walk.py

```diff
@@ -0,0 +1,28 @@
+import mgp
+import random
+
+
+@mgp.read_proc
+def get(
+    start: mgp.Vertex,
+    length: int = 10,
+) -> mgp.Record(path=mgp.Path):
+    """Generates a random path of length `length` or less starting
+    from the `start` vertex.
+
+    :param mgp.Vertex start: The starting node of the walk.
+    :param int length: The number of edges to traverse.
+    :return: Random path.
+    :rtype: mgp.Record(mgp.Path)
+    """
+    path = mgp.Path(start)
+    vertex = start
+    for _ in range(length):
+        try:
+            edge = random.choice(list(vertex.out_edges))
```

Comment:
Instead of loading all edges into context, an iterator should be created, that would iterate a random number of times up to the number of edges.